pyinjective/constant.py

```diff
@@ -78,36 +79,36 @@ def __init__(
     @classmethod
     def devnet(cls):
         return cls(
-            lcd_endpoint='devnet.lcd.injective.dev',
+            lcd_endpoint='https://devnet.lcd.injective.dev',
             grpc_endpoint='devnet.injective.dev:9900',
             grpc_exchange_endpoint='devnet.injective.dev:9910',
             chain_id='injective-777',
             fee_denom='inj',
-            env = 'devnet'
+            env='devnet'
         )
 
     @classmethod
     def testnet(cls):
         return cls(
-            lcd_endpoint='testnet.lcd.injective.dev',
-            grpc_endpoint='testnet-sentry0.injective.network:9900',
-            grpc_exchange_endpoint='testnet-sentry0.injective.network:9910',
+            lcd_endpoint='https://testnet.lcd.injective.dev',
+            grpc_endpoint='localhost:9900',
```

Comment:
Nam and Nikola recently added another sentry for testnet so we can include it here as a backup

Reply:
Oops, that's a typo, I changed to localhost only to test this out.